pex/venv/pex.py

```diff
@@ -440,9 +440,6 @@ def sys_executable_paths():
 
             os.environ["VIRTUAL_ENV"] = venv_dir
 
-            # A Python interpreter always inserts the CWD at the head of the sys.path.
-            sys.path.insert(0, "")
```

Comment:
I need to think through what PEX should be aiming for, but this is certainly not right no matter how that shakes out. In the zipapp execution mode, for example, "" is inserted, but only when the PEX runs in interpreter mode:
https://github.com/pantsbuild/pex/blob/67fc7e05315fc347bcf269712814f160c6035efe/pex/pex.py#L627-L631

Here in venv execution mode you've just wiped out "" insertion in all situations; so they don't behave the same.

Reply:
Ah I see, thanks for the code link. I'll dive further.